drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -74,6 +78,10 @@ struct spi_mcux_data {
 #endif
 };
 
+
+volatile bool reconfig;
```

Comment:
```suggestion
static volatile bool reconfig;
```
blocking PR for this issue